### PR TITLE
Create script to list files for bag in bagstore

### PR DESCRIPTION
**Pull request by lindareijnhoudt**
Fixes DD-

# Description of changes
* Add two help scripts copied from dans-datastation-tools
* Add script to list files for bag in bagstore

# How to test

# Related PRs

(Add links)

*

# Notify

@DANS-KNAW/dataversedans

### CODE REVIEW

File: src/easymigration/batch_processing.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    for pid in pids:
+        num += 1
+        logging.info("[{} of {}] Processing dataset with pid: {}".format(num, num_pids, pid))
+        try:
+            process_action_func(pid)
+        except Exception as e:
+            logging.exception("Exception occurred", exc_info=True)
+            if fail_on_first_error:
+                logging.error("Stop processing because of an exception:  {}".format(e))
+                break
```
Comment (jo-pol) [inline src/easymigration/batch_processing.py:19]: newbie question: does this break out of the exception handling or out of the loop

File: src/easymigration/scripts/list_bagstore_files.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    uuid = find_uuid(doi, dark_url)
+    files_xml = get_files_xml(uuid, dark_url)
+    parse_files_xml(doi, files_xml, csv_writer)
+
+
+def find_uuid(doi, bag_index_url):
+    #locate in bag-index
+    try:
+        params = {'doi': doi}
+        response = requests.put(
```
Comment (jo-pol) [inline src/easymigration/scripts/list_bagstore_files.py:25]: put to find something?

File: src/easymigration/scripts/list_bagstore_files.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            bag_index_url + '/bag-index/search',
+            params=params)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as re:
+        print("RequestException: ", re)
+        raise
+    resp_data = response.json()['result'][0]["bag-info"]["urn"]
```
Comment (jo-pol) [inline src/easymigration/scripts/list_bagstore_files.py:32]: What is the difference between these types of quotes?

File: src/easymigration/scripts/list_bagstore_files.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+def get_files_xml(urn, bag_store_url):
+    try:
+        url = "{}/bag-store/bags/{}/metadata/files.xml".format(bag_store_url, urn)
+        logging.debug(url)
+        response = requests.put(url)
+        response.raise_for_status()
+    except requests.exceptions.RequestException as re:
+        print("RequestException: ", re)
+        raise
```
Comment (jo-pol) [inline src/easymigration/scripts/list_bagstore_files.py:44]: just raise without an exception? Intersting discussion: https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python

**Review by jo-pol — COMMENTED** [review]
a first glance

**Comment by lindareijnhoudt** [comment]
taken over by #4

**Pull request closed without merging**